compiler/include/baseAST.h

```diff
@@ -146,6 +148,8 @@ typedef std::vector<NameAndSymbol> SymbolNameVec;
 // enumerated type of all AST node types
 //
 enum AstTag {
+  E_Thunk,
```

Comment:
I would expect it to be added next to TemporaryConversionSymbol in this enum rather than as the 1st thing

Reply:
Well, the thunk is an expr and the TCS is a symbol, so I was trying to keep them separated. I can definitely keep the thunk at the end of the expr / stmt list, which would put it very close to (barring a space) the TCS.

Reply:
Sounds good thanks